### PR TITLE
feat: add dashboard footer

### PR DESCRIPTION
**Pull request by BillySh1**
closes # mask-394
https://mask.atlassian.net/browse/MASK-394

### CODE REVIEW

**Comment by BillySh1** [comment]
@DimensionDev/qa

File: packages/dashboard/src/components/DashboardFrame/index.tsx [hunk 1]
```diff
@@ -44,6 +49,7 @@ export const DashboardFrame = memo((props: DashboardFrameProps) => {
                 {isLargeScreen && (
```
Comment (NikoManiac) [inline packages/dashboard/src/components/DashboardFrame/index.tsx:49]: It‘s still work on small screen?

File: packages/dashboard/src/components/DashboardFrame/index.tsx [hunk 2]
```diff
@@ -44,6 +49,7 @@ export const DashboardFrame = memo((props: DashboardFrameProps) => {
...
                     <LeftContainer item xs={2}>
                         <Navigation />
+                        <NavigationVersionFooter />
                     </LeftContainer>
                 )}
```
Comment (NikoManiac) [inline packages/dashboard/src/components/DashboardFrame/index.tsx:54]: It‘s still work on small screen?

**Comment by NikoManiac** [comment]
> closes # jira

Please fill Jira number.

File: packages/dashboard/src/components/DashboardFrame/index.tsx (hunk 2 above)
Comment (BillySh1) [reply packages/dashboard/src/components/DashboardFrame/index.tsx:54]: yes

**Comment by wenluomask** [comment]
Change menu bar's padding top to 28px in smaller window.
Screen Shot 2021-12-10 at 11 26 59 AM

**Review by Jack-Works — APPROVED** [review]

**Comment by BillySh1** [comment]
> Change menu bar's padding top to 28px in smaller window. Screen Shot 2021-12-10 at 11 26 59 AM

done please check again

**Comment by wenluomask** [comment]
Checked.

**Comment by wenluomask** [comment]
Link to Jira: https://mask.atlassian.net/jira/software/c/projects/MASK/boards/17?modal=detail&selectedIssue=MASK-390

File: packages/dashboard/src/components/NavigationVersionFooter/index.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    fontSize: '12px',
+}))
+
+interface NavigationVersionFooterProps extends React.PropsWithChildren<{}> {
+    version?: string
+}
+
+export const NavigationVersionFooter = memo((props: NavigationVersionFooterProps) => {
+    const t = useDashboardI18N()
+    const version = props.version ?? '2.2.0'
```
Comment (NikoManiac) [inline packages/dashboard/src/components/NavigationVersionFooter/index.tsx:18]: Why use 2.2.0 as default version?

**Review by NikoManiac — APPROVED** [review]

**Review by NikoManiac — APPROVED** [review]

File: packages/dashboard/src/components/NavigationVersionFooter/index.tsx (hunk 3 above)
Comment (BillySh1) [reply packages/dashboard/src/components/NavigationVersionFooter/index.tsx]: modified

**Review by Jack-Works — APPROVED** [review]

**Pull request merged**